maven-release-manager/pom.xml

```diff
@@ -161,6 +161,19 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.sonatype.sisu</groupId>
+      <artifactId>sisu-guice</artifactId>
+      <version>3.2.3</version>
+      <classifier>no_aop</classifier>
+      <scope>test</scope>
```

Comment:
Really this or the Guice from Google. This is a fork, no? @cstamas

Reply:
I left this one out since the actual dep is in provided scope and it does not matter at runtime.